opcua/server/uaprocessor.py

```diff
@@ -476,6 +476,38 @@ def _process_message(self, typeid, requesthdr, algohdr, seqhdr, body):
 
             self.send_response(requesthdr.RequestHandle, algohdr, seqhdr, response)
 
+        elif typeid == ua.NodeId(ua.ObjectIds.SetMonitoringModeRequest_Encoding_DefaultBinary):
+            self.logger.info("set monitoring mode request")
+
+            params = struct_from_binary(ua.SetMonitoringModeParameters, body)
+
+            # Send dummy results to keep clients happy
```

Comment:
thanks for the patch. but could just add #FIXME: at the beginning of comments so it gets flags av linters

Reply:
done in commit 465d703